docs/research/research_done_using_TARDIS/ads.ipynb

```diff
@@ -0,0 +1,918 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ads\n",
+    "import pandas as pd\n",
+    "ads.config.token = \"\" #Add your token"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "papers = ads.SearchQuery(q='(((full:\"tardis\" AND (full:\"kerzendorf\" OR (bibstem:\"Natur\" AND full:\"supernova\")))) AND year:2014-)+property:refereed', sort=\"date\",\n",
+    "                         fl = ['bibcode','title', 'bibstem', 'author', 'year'])\n",
+    "\n",
+    "bibcodes = []\n",
+    "titles = []\n",
+    "bibstems = []\n",
+    "authors = []\n",
+    "year_list = []\n",
+    "for paper in papers:\n",
+    "    bibcodes.append(paper.bibcode)\n",
+    "    titles.append(paper.title)\n",
+    "    bibstems.append(paper.bibstem)\n",
+    "    authors.append(paper.author)\n",
+    "    year_list.append(paper.year)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_url(bibcodes):\n",
+    "    \"\"\"This function takes the list of bibcodes and returns a link!\"\"\"\n",
+    "    url_list = []\n",
+    "    for bibcode in bibcodes:\n",
+    "        url = \"https://ui.adsabs.harvard.edu/abs/{}\".format(bibcode)\n",
+    "        url_list.append(url)\n",
+    "    return url_list\n",
+    "\n",
+    "def get_hyperlink(bibcodes):\n",
+    "    hyperlink_list=[]\n",
+    "    for url in get_url(bibcodes):\n",
+    "        hyperlink_list.append('`(ADS Link) <{}>`_'.format(url))\n",
+    "    return hyperlink_list"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_journal(bibstems):\n",
+    "    \"\"\"This function takes the list of bibstems and returns the journal that they are in, formatted.\"\"\"\n",
```

Comment:
You do not need to reformat the bibstem into a full journal name. TARDIS users will know what the journal abbreviations mean and can look them up via NASA ADS if not

Reply:
I understand that, but as we have already formatted the bibstems is it fine to leave it like this, or do you want it changed so it doesn't display the whole journal?